### DISCUSSION TITLE
Updated Plugin "Backup for Comicrack" to work with CE Edition

### DISCUSSION THREAD

**Discussion by sevymic** [maforget/ComicRackCE#29] [Announcements]
Hi
The plugin "Backup for Comicrack" don't works with CE edition because it's store the "%APPDATA%/cYo/ComicRack" folder and not the "%APPDATA%/cYo/ComicRack Community Edition"
Here is the link to the plugin : https://code.google.com/archive/p/cr-backup-manager/

Very useful plugin. It zip the ComicRack folder with ComicDB.xml each time CR starts. Can do a full backup too.

**Comment by maforget** [comment]
Yes I know, I mentioned it on Reddit already. But this isn't a ComciRackCE bug, this shouldn't be an Issue. There is nothing we can do, only the plugin author (or someone who wants to update it), should update it.

The plugin also doesn't work with portable installations because of that. The plugin author should have detected the location that the plugin starts in and work from there.  I've updated my local installation by just changing the path.

I will be closing it, but feel free to post it in the discussion section with a fixed version for others to get.

**Comment by maforget** [comment]
I've updated the plugin to support the Community Edition, Portable mode & Alternate Configurations.

https://github.com/maforget/cr-backup-manager/releases/latest

**Reply by maforget** [reply]
Just a FYI from anyone reading this in the future. This plugin isn't required anymore because the plugin functionality is built-in now. (See [Wiki](https://github.com/maforget/ComicRackCE/wiki/Backup-Manager))

**Comment by sevymic** [comment]
Thanks for the update.
Merci
